src/algebra/lie_algebra.lean

```diff
@@ -909,6 +930,10 @@ end
 def skew_adjoint_matrices_lie_subalgebra : lie_subalgebra R (matrix n n R) :=
 { lie_mem := J.is_skew_adjoint_bracket, ..(skew_adjoint_matrices_submodule J) }
 
+@[simp] lemma mem_skew_adjoint_matrices_lie_subalgebra (A : matrix n n R) :
+  A ∈ skew_adjoint_matrices_lie_subalgebra J ↔ A ∈ skew_adjoint_matrices_submodule J :=
+by refl
```

Comment:
Does `iff.rfl` work here? This would be slightly clearer.

Reply:
It does; thanks, this is better.